core-bundle/src/Twig/Finder/Finder.php

```diff
@@ -249,7 +249,7 @@ static function (array $chain) {
                 continue;
             }
 
-            if ($this->excludePartials && 1 === preg_match('%^.*_[^/]+$%', $identifier)) {
+            if ($this->excludePartials && 1 === preg_match('%[^/]+/_[^/\s]+$%', $identifier)) {
```

Comment:
```suggestion
            if ($this->excludePartials && 1 === preg_match('%(?:/|^)_[^/]+$%', $identifier)) {
```

What do you think about this one? The first non-capturing group directly in front of the underscore matches either a `/` or the start of the string. This way `_foo` and `foo/_bar` are both matched while `foo/bar_baz` won't..

Reply:
Yes, looks good to me as far as I tested 👍